src/lib/stack/_mixins.scss

```diff
@@ -0,0 +1,44 @@
+@use '../theme';
+@use './variables';
+
+@mixin core-styles() {
+  .forge-stack {
+    @include base();
+    @include theme.css-custom-property(gap, --forge-stack-gap, variables.$gap);
+    @include theme.css-custom-property(height, --forge-stack-height, 100%);
+  }
+}
+
+@mixin base() {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+@mixin inline() {
+  display: flex;
+  flex-direction: row;
+  --forge-stack-height: auto;
+}
+
+@mixin wrap() {
+  flex-wrap: wrap;
+}
+
+@mixin stretch() {
```

Comment:
I'm curious if it would be possible to enable "tracks" to stretch instead of individual items, so items in the bottom row are the same size as the ones above instead of filling the entire width. That would probably mean switching to `grid`... is that too big an ask?

Reply:
That in addition to the way it's handled now. just as an extra option

Reply:
I believe the intention is to keep this one dimensional. Nick and I discussed this potentially using `grid` to allow for multi dimensional layouts but I think it goes beyond the scope of what this basic layout component is intended for. Whether we use `grid` or `flex` probably doesn't matter much unless we get better responsive support with `grid`?

Reply:
`grid` is definitely capable of more complex responsiveness but you lose some perks like independently sized items. There's a tradeoff for sure. Some of the downsides to stretching everything could be mitigated by creatively setting the width of items.